cassiopeia/solver/__init__.py

```diff
@@ -9,7 +9,7 @@
 from .MaxCutGreedySolver import MaxCutGreedySolver
 from .MaxCutSolver import MaxCutSolver
 from .NeighborJoiningSolver import NeighborJoiningSolver
-from .solver_utilities import collapse_tree, collapse_unifurcations
+from .solver_utilities import collapse_tree
```

Comment:
Would it perhaps be more reasonable to add `collapse_tree` to the `CassiopeiaTree` class as well?

Reply:
I was actually thinking of this! But I'd like to save this for a future PR perhaps.

Reply:
Sounds good! Can you make a note in the `CassiopeiaTree` class?